### PR TITLE
User configuration page + Redesign Home menus

### PR DESCRIPTION
**Pull request by andreldsa**

### CODE REVIEW

File: app/auth/configProfileController.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+(function() {
+    var app = angular.module("app");
+    
+    app.controller("ConfigProfileController", function ConfigProfileController($state, InstitutionService, 
+            AuthService, UserService, $rootScope, $mdToast) {
+        var configPrileCtrl = this;
```
Comment (mayzabeel) [inline app/auth/configProfileController.js]: configProfileCtrl

File: app/auth/configProfileController.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+        configPrileCtrl.newUser = {};
+
+        Object.defineProperty(configPrileCtrl, 'user', {
```
Comment (mayzabeel) [inline app/auth/configProfileController.js]: configProfileCtrl

File: app/main/main.html [hunk 3]
```diff
@@ -5,43 +5,55 @@
       <md-icon>menu</md-icon>
     </md-button>
 
-    <md-button disabled="true">
-      <img src="images/oms.png" aria-label="WHO Logo" style="height: 80%;filter: brightness(10) grayscale(1);">
-    </md-button>
-    <h2>e-CIIS</h2>
+    <a ui-sref="app.home"layout="row" layout-align="center center">
+      <md-button disabled="true">
+        <img src="images/oms.png" aria-label="WHO Logo" style="height: 80%;filter: brightness(10) grayscale(1);">
```
Comment (mayzabeel) [inline app/main/main.html]: ajeita esse aria-label

**Comment by andreldsa** [comment]
Choose institution and Profile Configuration for new users #90

**Pull request merged**